dev-haskell/snap/snap-1.1.3.1-r1.ebuild

```diff
@@ -18,15 +18,13 @@ SLOT="0/${PV}"
 KEYWORDS="~amd64 ~x86"
 IUSE=""
 
-RESTRICT="test" # requires network access
```

Comment:
Was this intentionally removed? Does the test suite no longer require network access?

Reply:
Yes this was intentional, though I will double-check and then add a comment to the commit

Reply:
Nope, it was an accident. I'll put it back in, nice catch.